.github/workflows/fast-dispatch-full-regressions-and-models-impl.yaml

```diff
@@ -155,7 +155,7 @@ jobs:
         run: tar -xvf ttm_wormhole_b0.tar
       - uses: ./.github/actions/install-python-deps
       - name: Run frequent reg tests scripts
-        timeout-minutes: 50
+        timeout-minutes: 75
```

Comment:
do we still need this bump to 75 if we are skipping?
not quite sure why we are bumping if we are skipping some tests